builtin/settingtypes.txt

```diff
@@ -125,6 +125,10 @@ fixed_virtual_joystick (Fixed virtual joystick) bool false
 #    If enabled, virtual joystick will also tap "Aux1" button when out of main circle.
 virtual_joystick_triggers_aux1 (Virtual joystick triggers Aux1 button) bool false
 
+#    (Android) Use crosshair to select object instead of whole screen.
+#    If enabled, a crosshair will be shown and will be used for selecting object.
+use_crosshair (Use crosshair) bool false
```

Comment:
this should probably be `touch_use_crosshair` if it's specific to that

Reply:
I agree.